modules/bind-server/main.tf

```diff
@@ -27,73 +27,73 @@ resource "aws_instance" "bind" {
     no_device   = true
   }
 
-  tags {
-    Name = "${length(var.names) == 0 ? format("%s-%02d", var.name, count.index + 1) : var.names[count.index]}"
+  tags = {
+    Name = length(var.names) == 0 ? format("%s-%02d", var.name, count.index + 1) : var.names[count.index]
   }
 
   lifecycle {
-    ignore_changes = ["key_name"]
+    ignore_changes = [key_name]
```

Comment:
Is this right or should it be `var.key_name` ?

Reply:
I think it should be `"key_name"`, good catch.